### PR TITLE
Add new parser - Wfuzz 

### PR DESCRIPTION
**Pull request by SPoint42**
Wfuzz is an url fuzzer who can produce a JSON report.
It could be found at : https://github.com/xmendez/wfuzz

### CODE REVIEW

**Comment by SPoint42** [comment]
This is a WIP

File: dojo/fixtures/test_type.json [hunk 1]
```diff
@@ -719,6 +719,13 @@
         },
         "model": "dojo.test_type",
         "pk": 194
+    },
+    {
+        "fields": {
+            "name": "Wfuzz JSON Report"
+        },
+        "model": "dojo.test_type",
+        "pk": 195
```
Comment (damiencarol) [inline dojo/fixtures/test_type.json:728]: you don't need this modification anymore

Comment (SPoint42) [reply dojo/fixtures/test_type.json:728]: Need to modify the new parser doc so to reflect this

**Review by valentijnscholten — APPROVED** [review]

**Review by damiencarol — CHANGES_REQUESTED** [review]

File: dojo/fixtures/test_type.json (hunk 1 above)
Comment (damiencarol) [inline dojo/fixtures/test_type.json:728]:
```suggestion
```

**Review by Maffooch — APPROVED** [review]
@damiencarol merge when you're ready

**Review by Maffooch — CHANGES_REQUESTED** [review]
Wait I keep forgetting we are in the middle of movingthex docs to this repo. 
Please add this parser to the list here `docs/content/integrations/import.md`

**Comment by SPoint42** [comment]
Will do tomorrow the docs

**Review by SPoint42 — COMMENTED** [review]
> Wait I keep forgetting we are in the middle of movingthex docs to this repo.
> Please add this parser to the list here `docs/content/integrations/import.md`

✅ Doc is push

**Review by damiencarol — APPROVED** [review]

**Comment by damiencarol** [comment]
@Maffooch could you re-review this pull request?

**Review by devGregA — APPROVED** [review]

**Comment by damiencarol** [comment]
dismissing @Maffooch request for changes as @SPoint42 made the modifications requested.

**Pull request merged**